packages/relay/src/services/ReportService.ts

```diff
@@ -280,6 +295,389 @@ export class ReportService {
             },
         ]
     }
+
+    async claimPositiveReputation(
+        req: express.Request,
+        res: express.Response,
+        db: DB
+    ) {
+        await this.claimReputation(
+            req,
+            res,
+            db,
+            ClaimMethods.CLAIM_POSITIVE_REP,
+            ReputationDirection.POSITIVE
```

Comment:
It should be based on the result of the report to decide that user can be claim positive or negative reputation instead of this flag.

Reply:
fixed